mod/groups/views/default/group/default.php

```diff
@@ -15,10 +15,32 @@
 	return;
 }
 
+$members_count = $entity->getMembers(['count' => true]);
+
+$imprint = [
+	[
+		'icon_name' => 'users',
+		'content' => elgg_echo('groups:members_count', [$members_count]),
+		'class' => 'elgg-listing-group-members',
+	],
+];
+
+if (!$entity->isPublicMembership()) {
+	$imprint[] = [
+		'icon_name' => 'lock',
+		'content' => elgg_echo('groups:closed'),
+		'class' => 'elgg-listing-group-membership elgg-state elgg-state-danger',
+	];
+}
+
 $icon = elgg_view_entity_icon($entity, 'small', $vars);
 
 $vars['content'] = $entity->briefdescription;
 $vars['handler'] = 'groups';
+$vars['byline'] = false;
```

Comment:
not sure about this one

Reply:
I don't think owner info is important in the summary view

Reply:
that's my point. I think owner information can be very useful in the summary/listing view ;)

Reply:
talked with @jeabakker about this... he can live with it :)